dat/outfits/core_hull/medium/rs_medium_cargo_hull.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<outfit name="Red Star Medium Cargo Hull">
+ <general>
+  <typename>Core Systems (Hull)</typename>
+  <slot prop="hull">structure</slot>
+  <size>medium</size>
+  <priority>7</priority>
+  <mass>140</mass>
+  <price>360000</price>
+  <description>The Red Star Organisation have developed a hull modification that provides both ample cargo space and protection for their own work and have now started selling it to others. Deciding "in for a penny, in for a pound", an electromagnetic shockwave absorbtion generator was added, typically this makes the hull rather pointy and scanner obvious but since the hull is already rather lumpy due to the cargo pods... on the other hand, for the price of some CPU power, missile jams, tracking zero-points and reduced secondary splash damage are all achieved making this hull rather more resiliant.</description>
+  <gfx_store>cargo_hull_m.webp</gfx_store>
+  <cpu>-64</cpu>
+ </general>
+ <specific type="modification">
+  <cargo>80</cargo>
+  <absorb>20</absorb>
+  <armour>180</armour>
```

Comment:
I feel like the armour stats are too similar to existing cargo hulls to make it worth the extra cost and less space, considering most pirates i have seen harass me use unguided missiles, beams, or some other weapons
This applies to the small cargo hull especially
Also i feel the current CPU values are quite off-putting for at least the small hull

Reply:
It's worth bearing in mind that they have a cargo percentage bonus meaning the only hulls that have the final amount of cargo these do is the less defensive S&Ks, also that they are decidedly more defensive than the S&Ks and even than the more basic Hulls. They are designed as specialist hulls similar to the Nexus stealth hull and the S&K ultralight hull so they won't be cheap, especially as the engineering needed to make a hull that esesentially bridges defensive and cargo at the same time is always going to be more expensive than one that does either... specifically also the time to lock is longer and absorbtion higher than standard armour hulls and for a cargo hauler CPU is a useless dump stat except for turrets, and even then the engine is the limit on weight and so higher level turrets that use m,ore CPU aren't really possible anyway.
This hull is designed for a trader who fills to the briom with cargo and just ignores attacks and tried to make it to the jump gate / spob before the pirates notice or kill him... before them this was nigh on impossible, now it is just difficult!
Also note I also tried submitting better cargo hulls and they were too good and were rejected (with good reason if we can get proper point defence turrets working too :)

Reply:
On that point, in a cargo ship i usually put a defensive hull with decent cargo space, and also fill the thing to the crim with cargo pods and it usually gives me a larger chance of survival plus good cargo space

Reply:
Sorry, not easy. For various reasons *ow - my ears* - that included - Discord is not very workable for me, constant moving workstations doesn't help either :/

With the standard hulls you don't get a percentage bonus on you cargo space so they are much less efficient... as I mentioned somewhere else, you can get 600 tonnes on a Rhino, 900 on a Mule and 5000 on a Zebra with the cargo hulls, plus cargo pods to the max.

I've been working on making pure traders more survivable but there is a lot to do with trade generally since there is little consistancy between population figures, the economy, the desire to not let the player massively impact those economies, the trade lanes and how to calculate them, the number and sizes of trade ships and how much of that we actually represent and what sort of interspob and interfactional trade levels we are looking at, but there is a discussion topic and a better pull request that already discusses a lot of these things... please feel free to help out :D